Added release notes and details for block list labels included in Forms 14.2.0-rc2. (#6616)
umbraco/UmbracoDocs: 14/umbraco-forms/developer/blocklistfilters.md

# Block List Labels

When working with the Block List editor, [the editor experience is enhanced](https://docs.umbraco.com/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/block-list-editor#editor-appearance) by defining a label for the appearance of the Block.

These labels can use [Umbraco Flavored Markdown (UFM)](https://docs.umbraco.com/umbraco-cms/reference/umbraco-flavored-markdown).

> An option that allows for display of a forms name - `umbFormName` - is available for use.

Reviewer: ```suggestion
An option is available to display a form's name - `umbFormName`.
```
